lightkurve/lightcurve.py

```diff
@@ -1070,6 +1072,64 @@ def estimate_cdpp(self, transit_duration=13, savgol_window=101,
         cdpp_ppm = np.std(mean) * 1e6
         return cdpp_ppm
 
+    def query_solar_system_objects(self, cadence_mask='outliers', radius=None, sigma=3, cache=True):
+        '''Query to find what solar system are in a lightcurve or targetpixel file.
+
+        This function relies on The Virtual Observatory Sky Body Tracker (skybot) service which can be found here:
+            http://vo.imcce.fr/webservices/skybot/
+
+        Parameters
+        ----------
+        cadence_mask : str or bool
+            mask in time to select which frames or points should be searched for SSOs.
+            Default "outliers" will search for SSOs at points that are `sigma` from the mean.
+            "all" will search all cadences. Pass a boolean array with values of "True"
+            for times to search for SSOs.
+        radius : optional, float
+            Radius to search for bodies. If None, will set a sensible value.
```

Comment:
Can you clarify `sensible value` a bit?  I think it would be good to summarize what this function does in a few more sentences at the top of this docstring as well, e.g. the fact that it searches within a certain radius at certain times etc.